src/components/app.jsx

```diff
@@ -9,7 +9,7 @@ class App extends Component {
     super();
     this.state = {
       view: 'view-cards',
-      cards: []
+      cards: JSON.parse(localStorage.getItem('flash-cards')) ? JSON.parse(localStorage.getItem('flash-cards')) : []
```

Comment:
This is usually shortened to use `||` so you don't have to parse the JSON twice.

```js
JSON.parse(localStorage.getItem('flash-cards')) || []
```

Reply:
Oh got it!